### ISSUE TITLE
Sonoff SNZB-02D support

### ISSUE THREAD

**Issue by Scan4u** [AlexxIT/SonoffLAN#1142]
Hello AlexxIT,
Today I got the new Sonoff product - temp&humidity sensor [SNZB-02D](https://sonoff.tech/product/gateway-and-sensors/snzb-2d/)
It is successfully connected and shows in HASS panel, but the values display range is too big (x100):
![зображення](https://user-images.githubusercontent.com/16424225/226582002-bc0f8994-3002-48bd-a2ea-642110d02c08.png)
The humidity need to be 39,60% and the temperature - 24,40 degree. 
Is it possible to fix this issue?

**Comment by Scan4u** [comment]
Here U go:

`{
  "home_assistant": {
    "installation_type": "Home Assistant Container",
    "version": "2023.3.6",
    "dev": false,
    "hassio": false,
    "virtualenv": false,
    "python_version": "3.10.10",
    "docker": true,
    "arch": "x86_64",
    "timezone": "Europe/Kiev",
    "os_name": "Linux",
    "os_version": "4.4.180+",
    "run_as_root": true
  },
  "hacs": {
      "version": "1.31.0",
      "requirements": [
        "aiogithubapi>=22.10.1"
      ]
    },
    "sonoff": {
      "version": "3.5.0",
      "requirements": [
        "pycryptodome>=3.6.6"
      ]
    }
  },
  "integration_manifest": {
    "domain": "sonoff",
    "name": "Sonoff",
    "config_flow": true,
    "documentation": "https://github.com/AlexxIT/SonoffLAN",
    "issue_tracker": "https://github.com/AlexxIT/SonoffLAN/issues",
    "codeowners": [
      "@AlexxIT"
    ],
    "dependencies": [
      "http",
      "zeroconf"
    ],
    "requirements": [
      "pycryptodome>=3.6.6"
    ],
    "version": "3.5.0",
    "iot_class": "local_push",
    "is_built_in": false
  },
  "data": {
    "version": "6935784",
    "cloud_auth": true,
    "config": {
      "username": "***",
      "password": "***",
      "reload": "always",
      "force_update": [
        "temperature",
        "power"
      ],
      "scan_interval": "00:05:00"
    },
    "options": {
      "mode": "auto",
      "debug": false
    },
    "errors": [],
    "device": {
      "uiid": 1771,
      "params": {
        "subDevId": "12956efeff10fb301771",
        "parentid": "10012498d1",
        "fwVersion": "1.0.9",
        "tempComfortUpper": "2700",
        "ASYNC": 2,
        "humidity": "3220",
        "humiComfortStatus": 1,
        "tempComfortLower": "1900",
        "humiComfortLower": "4000",
        "humiComfortUpper": "6000",
        "tempUnit": 0,
        "temperature": "2540",
        "tempComfortStatus": 0,
        "timeZone": 2,
        "battery": 100,
        "supportPowConfig": 1,
        "trigTime": "1679549818110"
      },
      "model": "SNZB-02D",
      "online": true,
      "local": null,
      "localtype": null,
      "host": null,
      "deviceid": "a4xxxxxxb9"
    }
  }
}`

**Comment by Scan4u** [comment]
No, it was added automatically after the Sonoff SNZB-02D registration to my local ZBBridge, which is added to the HASS. 
No manual changes have been made by me.

**Comment by AlexxIT** [comment]
https://github.com/AlexxIT/SonoffLAN/releases/tag/v3.5.1
